core/controllers/admin.py

```diff
@@ -134,6 +136,23 @@ def post(self):
             elif self.payload.get('action') == 'reload_collection':
                 collection_id = self.payload.get('collection_id')
                 self._reload_collection(collection_id)
+            elif self.payload.get('action') == 'generate_dummy_explorations':
+                num_dummy_exps_to_generate = self.payload.get(
+                    'num_dummy_exps_to_generate')
+                num_dummy_exps_to_publish = self.payload.get(
+                    'num_dummy_exps_to_publish')
+                if isinstance(num_dummy_exps_to_generate) != int:
+                    raise self.InvalidInputException('%s is not a number'
+                        % num_dummy_exps_to_generate)
+                elif isinstance(num_dummy_exps_to_publish) != int:
+                    raise self.InvalidInputException('%s is not a number'
+                        % num_dummy_exps_to_publish)
+                elif num_dummy_exps_to_generate < num_dummy_exps_to_publish:
+                    raise self.InvalidInputException('Generate count cannot
```

Comment:
When breaking the contents of (...) over multiple lines, have a line break after the '('.